workers/Dockerfile

```diff
@@ -1,5 +1,13 @@
+# based on Debian 11 Bullseye
 FROM python:3.8
```

Comment:
This comment isn't true for the future, isn't it? If you want to have Debian bullseye, you have to use `debian:bullseye` instead of `python:3.8`.

Reply:
I removed the Debian version from the comment.